core/shared/src/main/scala/zio/clock/package.scala

```diff
@@ -22,12 +22,14 @@ import java.util.concurrent.TimeUnit
 import zio.duration.Duration
 
 package object clock {
+
+  import java.time.DateTimeException
```

Comment:
Very minor but can we move this to the imports at the top of the file with the other imports from `java.time`?

Reply:
Ohhh yeah, sorry, didn't realize that